packages/fxa-settings/scripts/test-ci.sh

```diff
@@ -11,11 +11,13 @@ mkdir -p artifacts/tests
 yarn workspaces foreach \
     --verbose \
     --topological-dev \
+    --include 123done \
     --include browserid-verifier \
     --include fxa-auth-db-mysql \
     --include fxa-auth-server \
     --include fxa-content-server \
     --include fxa-profile-server \
+    --include fxa-payments-server \
```

Comment:
Why is the payments server needed here?  I don't see any payment process in the new test, and no payment is required for 123Done.

Reply:
Ah I see.  Never mind me 👍